net/frr/src/opnsense/service/templates/OPNsense/Quagga/ospfd.conf

```diff
@@ -47,6 +47,9 @@ interface {{ physical_interface(interface.interfacename) }}
 {% endif %}
 !
 router ospf
+{% if helpers.exists('OPNsense.quagga.ospf.logadjacencychanges') and OPNsense.quagga.ospf.logadjacencychanges == '1' %}
```

Comment:
just a small note, knowing the rest of the file is the same, but this can be shortened with :  
```
{% if not helpers.empty('OPNsense.quagga.ospf.logadjacencychanges') %}
```